pyproject.toml

```diff
@@ -35,7 +35,7 @@ click = "^8.0"
 python-dotenv = "^1.0.0"
 rich = "^13.5.2"
 tabulate = "*"
-pick = "^2.2.0"
+pick = {version = ">=2.2.0,<3.0.0", optional = true, markers = "sys_platform != 'win32'"}
```

Comment:
forgot to include `winpty`?

Reply:
winpty was added later, as I wanted to use conpty stuff for shell stuff, will fix this later